metabase/metabase
# fix hidden pivot table scroll bar in QB

Closes https://github.com/metabase/metabase/issues/47595

### Description

Caused by https://github.com/metabase/metabase/pull/47102 specifically the part that attempted to address https://github.com/metabase/metabase/issues/47019 which requires hiding scroll bars in the dashboard editing mode. By a mistake scrollbars were hidden in all editing modes including QB.

### How to verify

Enable scrollbars visibility:
<img width="412" alt="Screenshot 2024-09-04 at 2 12 34 PM" src="https://github.com/user-attachments/assets/d2b5d227-94a5-489f-af7a-f8c026d85502">

- Create a Pivot table in QB
- Ensure scroll bars are visible (they were hidden but scroll with trackpad gestures or mouse still worked, the issue is with visibility)

### Checklist
